### DISCUSSION TITLE
Move specified running apps to current space in float layout by hotkey

### DISCUSSION THREAD

**Discussion by snakewa** [asmvik/yabai#789] [Q&A] [answered]
Hi guys, I am new to yabai, took few days to try others' dotfile. try to setup my shkd and yabai work together and setup stackline. Bascily I have surface undering about yabai's concept. 

Actually I have a need. Some application specially social / IM app like: whatsapp/telegram/messager/slack ... My workflow is that I work and from time to time to check if any new messages. So I really want to put them all in one space next to my primary space, so I could from time to time to check them my switching space. 

some questions:
1. how to move running app to current space
2. Since current space is tiled layout, so if use hotkey to call them -  move those apps and make them floating and check the messages. then I could use hotkey to move they to the space next to current one or put them tiled layout in the next space.

Looking for any sharing similar to my workflow so I could modify and learn from it

Thanks!

**Comment by joshtwo** [comment, marked as answer]
1. I'd do something like this:
```
for wid in $(yabai -m query --windows | jq '.[] | select(.app == "Skype") | .id'); do yabai -m window $wid --space mouse; done
```
To break this down:
* Get all the window ids of my chosen application with Yabai and jq (in this case, I'm testing on Skype).
* Move them all to the currently focused workspace (`mouse`).
2. You can basically do the same as above, except pick `next` instead of `mouse` as the workspace, then set `--toggle float` on each window.
